plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -244,6 +252,10 @@ private RowId getRowId(int position)
                                 long row = BIGINT.getLong(page.getBlock(ROW_ID_INDEX), i);
                                 RowId rowId = new RowId(originalTransaction, bucket, statement, row);
                                 deletedRowsBuilder.add(rowId);
+                                rowCount++;
+                                if (rowCount % 1000 == 0) {
```

Comment:
add a comment why

Reply:
dropped as data is not passed to operator context immediately.